shared/chat/conversation/input-area/normal/platform-input.desktop.tsx

```diff
@@ -186,6 +188,12 @@ class _PlatformInput extends React.Component<PlatformInputPropsInternal, State>
         onKeyDown={this._globalKeyDownPressHandler}
         onKeyPress={this._globalKeyDownPressHandler}
       >
+        {this.props.suggestBotCommandsUpdateStatus !== undefined &&
```

Comment:
does this not type w/o the `!== undefined` ? it should work just doing a truthy check ( `this.props.suggestBotCommandsUpdateStatus && `

Reply:
What is the best way to handle this if the value can be `0`?

Reply:
That is why I did it like this, the truthy check fails if this thing has the value 0.

Reply:
oh i see, didn't realize thats a valid value, k